src/servers/graphql-old-server.ts

```diff
@@ -11,6 +11,7 @@ import { and, shield } from "graphql-shield"
 import { makeExecutableSchema } from "graphql-tools"
 import moment from "moment"
 import path from "path"
+import { takeRight } from "lodash"
```

Comment:
I'd like to eventually get rid of lodash, so please do:

`import takeRight from "lodash.takeright"`